styles/components/newsfeed.css

```diff
@@ -0,0 +1,46 @@
+.content-container {
+    height: 140px;
```

Comment:
Please find a way to avoid the fixed width. What if someone has a bigger font size?